azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/ReceiverRuntimeInformation.java

```diff
@@ -10,76 +10,40 @@
  * Represents the temporal receiver runtime information for a {@link PartitionReceiver}.
  * Current received {@link EventData} and {@link ReceiverRuntimeInformation} can be used to find approximate value of pending events (which are not processed yet).
  */
-public final class ReceiverRuntimeInformation {
-
-    private final String partitionId;
-
-    private long lastSequenceNumber;
-    private Instant lastEnqueuedTime;
-    private String lastEnqueuedOffset;
-    private Instant retrievalTime;
-
-    public ReceiverRuntimeInformation(final String partitionId) {
-
-        this.partitionId = partitionId;
-    }
+public interface ReceiverRuntimeInformation {
 
     /**
      * Get PartitionId of the {@link PartitionReceiver} for which the {@link ReceiverRuntimeInformation} is returned.
      *
      * @return Partition Identifier
      */
-    public String getPartitionId() {
-
-        return this.partitionId;
-    }
+    String getPartitionId();
 
     /**
      * Get sequence number of the {@link EventData}, that is written at the end of the Partition Stream.
      *
      * @return last sequence number
      */
-    public long getLastSequenceNumber() {
-
-        return this.lastSequenceNumber;
-    }
+    long getLastSequenceNumber();
```

Comment:
>getLastSequenceNumber [](start = 9, length = 21)

copy-pasted from .net
Should this have been getLast**Enqueued**SequenceNumber ???? copy-pasted from .net